app.py

```diff
@@ -144,75 +158,97 @@ async def create_directory():
 @app.post("/inf")
 async def inference_request():
     """
-    performs inference on an image, and returns the results.
-    The image and inference results uploaded to a folder in the user's container.
-    """
+    Performs inference on an image, and returns the results.
+    The image and inference results are uploaded to a folder in the user's container.
+    """  
+     
+    seconds = time.perf_counter() # transform into logging
     try:
+        print("Entering inference request") # Transform into logging
         data = await request.get_json()
-        connection_string: str = os.environ["NACHET_AZURE_STORAGE_CONNECTION_STRING"]
+        pipeline_name = data.get("model_name", "defaul_mode")
```

Comment:
default_mode?

Reply:
Remove it, since frontend seems to only return `model_name`